website/docs/reference/global-configs/behavior-changes.md

```diff
@@ -152,3 +154,13 @@ on-run-start:
   - '{{ ... if flags.WHICH != 'freshness' }}'
 ```
 </File>
+
+
+### MetricFlow time spine YAML
+The `require_yaml_configuration_for_mf_time_spines` flag is set to `False` by default.
+
+In previous versions (dbt Core 1.8 and earlier), the MetricFlow time spine configuration was stored in a `metricflow_time_spine.sql` file.
+
+When the flag is set to `True`, dbt will continue to support the SQL file configuration. When the flag is set to `False`, dbt will raise a deprecation warning if it detects a MetricFlow time spine configured in a SQL file. 
+
+The MetricFlow YAML file should be named `models/_models.yml` and should be located in the `models` directory.
```

Comment:
I think the model can be named anything as long as it has the `time_spine:` field. Maybe we just link to the v1.9 docs [here](https://docs.getdbt.com/docs/build/metricflow-time-spine)?